src/components/CorporateCampaign/CampaignProgressBar.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+	<div
+		class="tw-rounded tw-bg-brand-50 tw-w-full tw-py-1 tw-px-1 tw-mb-2 tw-mt-2"
+	>
+		<template v-if="promoAmount">
+			<h4 class="tw-mt-2 tw-mb-1 tw-px-4">
+				Let's Get Started
+			</h4>
+			<h3 class="tw-mb-2 tw-px-4">
+				You have ${{ promoAmount | numeral }} in credit from {{ promoName }}
```

Comment:
Looks like `promoName` isn't making it to this component currently, potentially related to the comment by @michelleinez on `CCLandingPage`